tests/test_meta_sqlite.py

```
@@ -118,22 +118,22 @@ def test_metastore_setup_wrong_db_type(self):
         )()
 
         with patch(
-            "src.synapso_core.persistence.implementations.meta_store.meta_sqlite.get_config",
+            "synapso_core.data_store.backend.sqlite.sqlite_meta_store.get_config",
             return_value=mock_config,
         ):
             with pytest.raises(ValueError, match="Meta store type is not sqlite"):
-                adapter = MetaSqliteAdapter()
+                adapter = SqliteMetaStore()
```

Ruff (0.12.2)

Local variable `adapter` is assigned to but never used

Remove assignment to unused variable `adapter`

(F841)